cgi-bin/LJ/User/Administration.pm

```diff
@@ -329,6 +329,22 @@ sub unban_user_multi {
 
 
 ########################################################################
+### Selective Screening functions
+
+# return if $target's comments will automatically be screened in $u's journal
+sub has_autoscreen {
+    my ( $u, $target ) = @_;
+
+    my $uid = LJ::want_userid( $u );
+    my $jid = LJ::want_userid( $target );
+    return 1 unless $uid && $jid;
```

Comment:
Sorry for waiting so long for another round of review.  This one line just jumped out at me as problematic: if $jid is undefined (as it would be for logged-out comments), shouldn't this method return 0 and not 1?  Otherwise we will end up always screening logged-out comments, which is... not what we want.